### PR TITLE
Updated wx rotating cube example

### PR DESCRIPTION
**Pull request by cansik**
This PR updates the [cube_wx.py](https://github.com/pygfx/pygfx/blob/main/examples/other/cube_wx.py) example to be compatible with the most recent version of pygfx (world / local position, readonly vectors and so on). To be able to test it, I had to include the wgpu package, where the following PR has been applied: https://github.com/pygfx/wgpu-py/pull/486

Now everything works as expected (currently tested on MacOS):

image

### CODE REVIEW

**Comment by almarklein** [comment]
Thanks! I think we need to hold this until your PR @ wgpu lands in a release. But this looks good!

**Comment by almarklein** [comment]
This is ready to go!

**Review by Korijn — APPROVED** [review]

**Pull request merged**
